tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- needs to contain all frameworks which src projects wish to restore -->
+    <TargetFramework Condition="'$(UseIntrinsics)' != 'true'">netstandard2.0</TargetFramework>
+    <TargetFrameworks Condition="'$(UseIntrinsics)' == 'true'">netstandard2.0;netcoreapp3.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="1.0.0" />
+    <PackageReference Include="Microsoft.ML.DataView" Version="1.0.0" />
+    <PackageReference Include="Microsoft.ML.CpuMath" Version="1.0.0" />
+    <PackageReference Include="Microsoft.ML.FastTree" Version="1.0.0" />
+    <PackageReference Include="Microsoft.ML.LightGbm" Version="1.0.0" />
+    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.0.0" />
+    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.0.0" />
+  </ItemGroup>
+
+  <!-- The purpose of this target is to return a path from a referenced
+       package / project -->
+  <Target Name="GetContract" DependsOnTargets="ResolveReferences" Returns="@(_contractReferencePath)">
+    <Error Condition="'$(ContractName)' == ''" Text="ContractName must be specified" />
+    <ItemGroup>
+      <_contractReferencePath Include="@(ReferencePath)" Condition="'%(FileName)' == '$(ContractName)'" />
+      <_allReferenceDirectories Include="%(ReferencePath.RootDir)%(ReferencePath.Directory)" />
+      <_contractReferencePath Include="@(ReferencePath)" Condition="'%(FileName)' == '$(ContractName)'" />
+      <_contractReferencePath DependencyPaths="@(_allReferenceDirectories)" />
+    </ItemGroup>
+    <Error Condition="'@(_contractReferencePath)' == ''" Text="Could not locate $(ContractName)" />
+  </Target>
+
+  <Target Name="Build">
```

Comment:
This could be improved by changing the project extension and defining a couple targets.  We do that elsewhere: 
https://github.com/dotnet/project-system/issues/4647

IOW: call this a .proj, or a .restoreproj or something.  SLN entry looks the same (you have to do it manually in text editor, IDE won't let you).  Add the workaround I linked, define stub targets for anything that fails, and it should avoid 
the confusion around a CSProj that doesn't build anything.
You don't need to do that now, but I think it would make this more sensible.

Reply:
I have tried to add the lines found in the issue to the new .restoreproj file but that did not work. 
I will keep it as is for now then.